config/project/entryTypes/person--ad5b3d91-425f-4181-8ded-f9c7c53a6bba.yaml

```diff
@@ -0,0 +1,244 @@
+color: null
+fieldLayouts:
+  51030e30-ba35-4a66-ae4c-49cb4bdddc1c:
+    tabs:
+      -
+        elementCondition: null
+        elements:
+          -
+            autocapitalize: true
+            autocomplete: false
+            autocorrect: true
+            class: null
+            dateAdded: '2025-02-25T04:11:28+00:00'
+            disabled: false
+            elementCondition: null
+            id: null
+            includeInCards: false
+            inputType: null
+            instructions: null
+            label: Name
+            max: null
+            min: null
+            name: null
+            orientation: null
+            placeholder: null
+            providesThumbs: false
+            readonly: false
+            required: true
+            size: null
+            step: null
+            tip: null
+            title: null
+            type: craft\fieldlayoutelements\entries\EntryTitleField
+            uid: 2fa68b7e-55c2-4890-97be-1592da7ae190
+            userCondition: null
+            warning: null
+            width: 100
+          -
+            dateAdded: '2025-02-25T04:17:41+00:00'
+            elementCondition: null
+            fieldUid: 840acb32-6694-40d0-9384-60a0220c1de0 # Label
+            handle: designation
+            includeInCards: false
+            instructions: null
+            label: Designation
+            providesThumbs: false
+            required: false
+            tip: null
+            type: craft\fieldlayoutelements\CustomField
+            uid: 5eaa009e-fd0a-4dc6-9def-4dc6dfe0e9c4
+            userCondition: null
+            warning: null
+            width: 50
+          -
+            dateAdded: '2025-02-25T04:17:41+00:00'
+            elementCondition: null
+            fieldUid: 69339afc-d4f1-440b-a7ad-7da2ccb77972 # Image
+            handle: photo
+            includeInCards: false
+            instructions: null
+            label: Photo
+            providesThumbs: false
+            required: false
+            tip: null
+            type: craft\fieldlayoutelements\CustomField
+            uid: 6b80c15b-e48e-4b66-8f5a-795142b9a77f
+            userCondition: null
+            warning: null
+            width: 50
+          -
+            dateAdded: '2025-02-25T07:46:41+00:00'
+            elementCondition: null
+            fieldUid: 856911a9-89f9-4865-bdb1-5665460c6c23 # Contact Information
+            handle: null
+            includeInCards: false
+            instructions: null
+            label: null
+            providesThumbs: false
+            required: false
+            tip: null
+            type: craft\fieldlayoutelements\CustomField
+            uid: bc7fb77c-d866-42b6-a1fb-401c268161e3
+            userCondition: null
+            warning: null
+            width: 100
+          -
+            dateAdded: '2025-02-25T09:26:43+00:00'
+            elementCondition: null
+            fieldUid: 52ed9e0d-6fb7-457a-b237-fa5bab3271f6 # Social Media
```

Comment:
If we go with a matrix field for Social media as well, then we can perhaps have a single field that has entry type for Contact as well as Social media clubbed together. For other data models where only social media or only contact might make sense, we can create matrix fields with subsets of those entry types.